server/src/game_player.go

```diff
@@ -189,15 +191,14 @@ func (p *GamePlayer) PlayCard(c *Card) {
 	// Update the progress
 	progressFloat := float64(g.Score) / float64(g.MaxScore) * 100 // In percent
 	progress := int(math.Round(progressFloat))
-	oldProgress := t.Progress
```

Comment:
what does this have to do with logging?

Reply:
That's a fix from linter warnings (ifshort)